installation/local/bootstrap.sh

```diff
@@ -213,6 +238,14 @@ export PYTHONPATH=\${PYTHONPATH}
 echo \$PATH | grep ${CO_SIM_NEST}/bin 1>/dev/null 2>&1
 [ \$? -eq 0 ] || export PATH=$CO_SIM_NEST/bin:\${PATH}
 
+# start NEST Desktop
+export NEST_DESKTOP_HOST=0.0.0.0
+nest-desktop start &
+
+# start Insite Access node
+insite-access-node &
```

Comment:
@marcelkrueger Could you check this and - if necessary - add the missing lines for the Insite start?

(see also https://github.com/multiscale-cosim/Cosim_NestDesktop_Insite/pull/22#issuecomment-1702429428)